yt_dlp/extractor/unsupported.py

```diff
@@ -50,6 +50,7 @@ class KnownDRMIE(UnsupportedInfoExtractor):
         r'music\.amazon\.(?:\w{2}\.)?\w+',
         r'(?:watch|front)\.njpwworld\.com',
         r'qub\.ca/vrai',
+        r'crunchyroll\.com',
```

Comment:
prob isn't necessary, but
```suggestion
        r'(?:beta\.)?crunchyroll\.com',
```

?

regardless we should add a test (or tests)